x/ibc-account/keeper/relay.go

```diff
@@ -40,7 +40,13 @@ func (k Keeper) TryRegisterIBCAccount(ctx sdk.Context, sourcePort, sourceChannel
 		Data: salt,
 	}
 
-	// TODO: Add timeout height and timestamp
+	timeoutHeight := clienttypes.Height{
+		RevisionNumber: 2,
+		RevisionHeight: ^uint64(0),
+	}
```

Comment:
```suggestion
	timeoutHeight := clienttypes.ZeroHeight()
```

Reply:
also good to add a comment explaining that a zero timeout height disables the usage of this timeout value

Reply:
Yep. Updated :+1: